### ISSUE TITLE
build-scaleway-ubuntu1604-x64-2 out of space

### ISSUE THREAD

**Issue by sxa555** [AdoptOpenJDK/openjdk-infrastructure#1192]
@dinogun, this seems to be due to docker images which are taking up about 33Gb. Is this expected? If so we need to do some additional capacity planning on [that box](https://ci.adoptopenjdk.net/computer/build-scaleway-x64-ubuntu-16-04-2/) as it's currently filling up the root filesystem

**Comment by dinogun** [comment]
@sxa555 While the docker job uses a lot of space ... it should be a temporary surge and it cleans up several times during the run to ensure we dont run into out of space issues. However will check to see if something new is pushing it beyond the limit.

**Comment by sxa555** [comment]
It's still looking to be at 100% on that filesystem - any obvious sign of why?

**Comment by sxa555** [comment]
I've moved `/var/lib/docker` to `/home/docker/var-lib-docker` and symlinked it back to the old location (I tried adding an extra filesystems but they didn't allow that to work with the machine type that we have). That shoudl alleviate the immediate issue, but we do need to understand and resolve the issue of why we have so much seemingly being left around between runs. I'll leave this open for now until we have that understood.
